### DISCUSSION TITLE
Scheduler delete unwanted Models

### DISCUSSION THREAD

**Discussion by jud3v** [laravel/framework#34521] [General]
- Laravel Version: 8.1.0
- PHP Version: 
   production: 7.3,
   local: 7.4.10
- Database Driver & Version: 
   production: Ver 15.1 Distrib 10.3.23-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
   local: Ver 15.1 Distrib 10.4.14-MariaDB, for Linux (x86_64) using  EditLine wrapper
### Description:

Every day at 01:00 some models will be deleted and i don't know why because in my code i don't have any things that delete any models,

exemple i have 9 required scheduler for my apps and only one can delete the Product models. but no other models like Category, Topic, Users.

it's been two days since this happens to me would you have a solution for me? the only effective one that I have found for the moment these to suspend the scheduler and make backup every day.

### Steps To Reproduce:

update laravel 7.25.0 to 8.1.0 wait scheduler unwanted models will be deleted for my case.

**Comment by taylorotwell** [comment]
I can promise that Laravel does not delete models on its own initiative.
